MVP Module/Source/Feed Presentation/FeedPresenter.swift

```
@@ -13,13 +13,19 @@ protocol FeedView {
 	func display(_ viewModel: FeedViewModel)
 }
 
+protocol FeedErrorView {
+	func display(_ viewModel: FeedErrorViewModel)
+}
+
 final class FeedPresenter {
 	private let feedView: FeedView
 	private let loadingView: FeedLoadingView
+	private let errorView: FeedErrorView
 	
-	init(feedView: FeedView, loadingView: FeedLoadingView) {
+	init(feedView: FeedView, loadingView: FeedLoadingView, errorView: FeedErrorView) {
 		self.feedView = feedView
 		self.loadingView = loadingView
+		self.errorView = errorView
 	}
 	
 	func didStartLoadingFeed() {
```

The Presenter can decide when to show/hide the message. 

For example, in the `didStartLoadingFeed()` method, the Presenter can call  `loadingView.display(FeedErrorViewModel(errorMessage: .none)` to hide the error.

This way, the ViewController doesn't need to decide when to hide the error. It's much easier to manage presentation state when all changes are centralized in the presenter.